plat/rockchip/rk3399/drivers/pmu/rk3399m0.h

```diff
@@ -0,0 +1,10 @@
+#ifndef __RK3399M0_H__
```

Comment:
Needs copyright boiler-plate info here.
This would need to be addressed ahead of a merge.